src/kernels/jupyter/session/jupyterSession.ts

```diff
@@ -334,6 +339,9 @@ export class JupyterSession extends BaseJupyterSession {
                         if (this.connInfo && backingFile) {
                             this.contentsManager.delete(backingFile.path).ignoreErrors();
                         }
+                        if (backingFile) {
+                            this.fs.deleteLocalFile(backingFile.path).ignoreErrors();
```

Comment:
Jupyter CLI doesn't provide any argument to disable checkpoints.
We'd have to update the config files, and thats a lot more work (in case users have custom configs, etc) than just deleting manually.